bundles/org.openhab.binding.radiothermostat/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -6,7 +6,7 @@
 
 	<!-- RadioThemostat Thing -->
 	<thing-type id="rtherm">
-		<label>Thermostat</label>
+		<label>CT30, CT50/3M50 or CT80 Wi-Fi Thermostat</label>
```

Comment:
Labels are kept short intentionally:

See: https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

> The label should be short so that for most UIs it does not spread across multiple lines. Guideline is 2-3 words with up to 25 chars. Labels should be capitalized using the following rules:

Reply:
Ditto the comment from #11674